components/multi-platform-controller/staging/host-config.yaml

```diff
@@ -7,35 +7,211 @@ metadata:
   namespace: multi-platform-controller
 data:
 
-  dynamic-platforms: linux/arm64,linux/amd64,linux-root/arm64,linux-root/amd64
+  # dynamic-platforms: linux/arm64,linux/amd64,linux-root/arm64,linux-root/amd64
+  dynamic-platforms: linux-mlarge/amd64,linux-mlarge/arm64,linux-mxlarge/amd64,linux-mxlarge/arm64,linux-m2xlarge/amd64,linux-m2xlarge/arm64,linux-m4xlarge/amd64,linux-m4xlarge/arm64,linux-m8xlarge/amd64,linux-m8xlarge/arm64,linux-cxlarge/amd64,linux-cxlarge/arm64,linux-c2xlarge/amd64,linux-c2xlarge/arm64,linux-c4xlarge/amd64,linux-c4xlarge/arm64,linux-c8xlarge/amd64,linux-c8xlarge/arm64,linux-root/arm64,linux-root/amd64
```

Comment:
The way users select the platform/vm size is by using of the  values from this list in their pipeline definition. If we remove the 2 platforms `linux/arm64`  and `linux/amd64`, this will be a breaking change.

I suggest to avoid the breaking change that we put back those 2 and consider them the default choice for multi platform build. We need to pick a VM size that will be the default, m6a-large (2vCPU/8GiB)

Reply:
default platform/vm (`linux/arm64 && linux/amd64`) back in place and removed `linux-mlarge/amd64 and linux-mlarge/arm64`.